core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/jvm/JVMType.java

```diff
@@ -71,8 +71,11 @@ public static JVMType getJVMType(String jvmName) {
 			return JVMType.JROCKIT;
 		} else if (JavaVMVersionToolkit.isHotspotJVMName(jvmName)) {
 			return JVMType.HOTSPOT;
+<<<<<<< HEAD
```

Comment:
This is where spotless wasn't too happy in that last run. There's two more to clean up below as well.